css/style.css

```diff
@@ -0,0 +1,99 @@
+
+body {
+  background-color: grey;
+  display: grid;
+  font-family: sans-serif;
+	font-size: 24px;
+	margin: 0;
```

Comment:
Indentation issue here

Reply:
This looks like a tabs vs. spaces issue - looks fine in VS Code, but not on GitHub, so you can disregard this.